WalletWasabi/Wallets/WalletFilterProcessor.cs

```diff
@@ -93,12 +96,11 @@ protected override async Task ExecuteAsync(CancellationToken cancellationToken)
 		{
 			while (true)
 			{
-				cancellationToken.ThrowIfCancellationRequested();
```

Comment:
@kiminuo This is the only change that I do not understand, can you explain it please?